Resources/Prototypes/Recipes/Lathes/machine_boards.yml

```
@@ -462,3 +462,8 @@
   parent: [ BaseCircuitboardRecipe, BaseGeneralMachineRecipeCategory ]
   id: HolopadMachineCircuitboard
   result: HolopadMachineCircuitboard
+
+- type: latheRecipe
+  parent: BaseGoldCircuitboardRecipe
```

This should inherit the parent for lathe categories, and be moved up to line 300 with the other cargo machine boards.

```suggestion
  parent: [ BaseGoldCircuitboardRecipe, BaseSupplyMachineRecipeCategory ]
```